packages/devtools_app/lib/src/screens/memory/panes/diff/controller/sampler.dart

```diff
@@ -71,11 +83,24 @@ class HeapClassSampler extends ClassSampler {
       ClassType.runtime;
 
   @override
-  Future<void> manyLiveToConsole() async {
-    serviceManager.consoleService.appendInstanceSet(
-      type: heapClass.shortName,
-      instanceSet: (await _liveInstances())!,
+  Future<void> allLiveToConsole({
+    required bool includeSubclasses,
+    required bool includeImplementers,
+  }) async {
+    final list = await _liveInstancesAsList();
+
+    if (list == null) {
```

Comment:
should we print this message if an error is thrown in _liveInstancesAsList as well?